test/Microsoft.AspNet.Mvc.FunctionalTests/TryValidateModelTest.cs

```diff
@@ -1,137 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net;
-using System.Net.Http;
-using System.Text;
-using System.Threading.Tasks;
-using Microsoft.AspNet.Testing;
-using Newtonsoft.Json;
-using Xunit;
-
-namespace Microsoft.AspNet.Mvc.FunctionalTests
-{
-    public class TryValidateModelTest : IClassFixture<MvcTestFixture<ValidationWebSite.Startup>>
```

Comment:
Remove corresponding code from the `ValidationWebSite`. Just dangling bloat now.